src/_locales/cs.json

```diff
@@ -0,0 +1,114 @@
+{
+  "audio-device-communications": "Vychozí komunikační zařízení",
+  "audio-device-default": "Výchozí systémové zařízení",
+  "audio-device-unknown": "Neznámé zařízení",
+
+  "button-text-ok": "OK",
+  "button-text-cancel-alarm": "Zrušit budík",
+  "button-text-click-here-to-set-alarm": "Klikněte zde pro nastavení budíku",
+  "button-text-close": "Zavřít",
+  "button-text-dont-tell-me-again": "Znovu neupozorňovat",
+  "button-text-lets-do-this": "Pojďme do toho!",
+  "button-text-lets-go": "Tak jdem na to!",
+  "button-text-not-now": "Ne nyní",
+
+  "label-about": "O Aplikaci",
+  "label-alarm": "Budík",
+  "label-desktop-settings": "Nastavení Desktop Přehrávače",
+  "label-donate": "Podpořit",
+  "label-help" : "Nápověda",
+  "label-issues": "Problémy s aplikací",
+  "label-learn-more": "Zjistit více",
+  "label-loading-devices": "Načítání zařízení...",
+  "label-pause-after-song": "Pozastavit po přehrání této skladby",
+  "label-quit": "Ukončit",
+  "label-show-lyrics": "Zobrazit text skladby (Beta)",
+  "label-unknown-artist": "Neznámý umělec",
+  "label-unknown-song": "Neznámá skladba",
+  "label-version": "Verze",
+
+  "lastfm-login-authorized": "Přihlášen",
+  "lastfm-login-authorizing": "Probíhá ověření",
+  "lastfm-login-button-text": "Přihlásit se do služby Last.FM",
+  "lastfm-login-error": "Vypadá to, že jste opustili přihlašovací proces. Pokud se tak nestalo, prosím, opakujte přihlášení",
+  "lastfm-login-not-authorized": "Nepřihlášen",
+  "lastfm-logout-button-text": "Odhlásit se ze služby Last.FM",
+
+  "lyrics-failed-message": "Omlouváme se, ale nenašly se žádné texty pro tuto skladbu",
+  "lyrics-loading-message": "Načítání textu skladby...",
+  "lyrics-no-song-message": "Právě nehraje žádná skladba",
+
+  "message-pausing-after-song": "Pozastavení po této skladbě.",
+  "message-pausing-after-song-button": "Nepozastavovat po této skladbě",
+  "message-uncaught-error": "Nastala nespecifikovaná chyba v programu (GPMDP).",
+  "message-uncaught-error-button": "Klikněte zde pro nahlášení tohoto problému na GitHub",
+
+  "modal-confirmTray-title": "Jen abyste věděli",
+  "modal-confirmTray-content": "Zavřením hlavního okna programu (to oč jste se právě pokusili) neukončí program, ale pouze jej minimalizuje do systémové lišty. Pokud chcete tuto možnost změnit, můžete tak učinit v okně \"Nastavení Desktop Přehrávače\".",
+  "modal-confirmOpenPort-content": "Vypadá to, že Váš firewall blokuje službu PlaybackAPI. Můžeme přidat výjimku za Vás, ale upozorňujeme, že to může vyžadovat administrátorské oprávnění",

```

Comment:
Pozměnil bych znění "Můžeme přidat výjimku za Vás, na to ale budeme nejspíše potřebovat administrátorské oprávnění" :) zní to tak 'lidštěji' :)